src/toil/job.py

```diff
@@ -254,6 +298,32 @@ def encapsulate(self):
         """
         return EncapsulatedJob(self)
 
+
+    @staticmethod
+    def update_kwargs(kwargs):
```

Comment:
If you modify kwargs in place you shouldn't return it. Returning it would indicate to the caller that you return a copy and that you didn't touch the argument, but the copy.